src/ui-shell/header/header-menu.component.ts

```diff
@@ -44,6 +57,15 @@ export class HeaderMenu {
 	}
 	@Input() trigger: "click" | "mouseover" = "click";
 
+	@Input() headerItems: HeaderItemInterface[];
```

Comment:
Maybe add some docs around this :+1: especially if this is the recommended usage model going forward